### ISSUE TITLE
Can I ignore this warning? UndefinedMetricWarning: Precision and F-score are ill-defined and being set to 0.0 in labels with no predicted samples. 

### ISSUE THREAD

**Issue by kkkMeng** [rungjoo/CoMPM#14]
I met the warning below:
/root/miniconda3/lib/python3.8/site-packages/sklearn/metrics/_classification.py:1471: 
UndefinedMetricWarning: Precision and F-score are ill-defined and being set to 0.0 in labels with no predicted samples. Use `zero_division` parameter to control this behavior.
  _warn_prf(average, modifier, msg_start, len(result))

But the accuracy still printed. And seems like the train can be continue. I am wondering will this affect the training?

**Comment by rungjoo** [comment]
Sorry for my belated reply.

This error occurs in the scikit-learn library.
This shows what happens when a 0 is entered in the denominator.

Please refer to the link below.
https://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html

You can ignore the warning.
